ios/NativeViews/TextileImageView.m

```diff
@@ -1,462 +1,91 @@
 #import "TextileImageView.h"
-
 #import <React/RCTBridge.h>
-#import <React/RCTConvert.h>
-#import <React/RCTEventDispatcher.h>
-#import <React/RCTUtils.h>
-#import <React/UIView+React.h>
-
-#import "RCTImageBlurUtils.h"
-#import <React/RCTImageLoader.h>
-#import "RCTImageUtils.h"
-
 #import "TextileNode.h"
-#import "TextileImageSource.h"
-
-/**
- * Determines whether an image of `currentSize` should be reloaded for display
- * at `idealSize`.
- */
-static BOOL RCTShouldReloadImageForSizeChange(CGSize currentSize, CGSize idealSize)
-{
-  static const CGFloat upscaleThreshold = 1.2;
-  static const CGFloat downscaleThreshold = 0.5;
-
-  CGFloat widthMultiplier = idealSize.width / currentSize.width;
-  CGFloat heightMultiplier = idealSize.height / currentSize.height;
-
-  return widthMultiplier > upscaleThreshold || widthMultiplier < downscaleThreshold ||
-  heightMultiplier > upscaleThreshold || heightMultiplier < downscaleThreshold;
-}
-
-/**
- * See RCTConvert (ImageSource). We want to send down the source as a similar
- * JSON parameter.
- */
-static NSDictionary *onLoadParamsForSource(TextileImageSource *source)
-{
-  NSDictionary *dict = @{
-                         @"width": @(source.size.width),
-                         @"height": @(source.size.height),
-                         @"hash": source.hsh,
-                         @"path": source.path,
-                         };
-  return @{ @"source": dict };
-}
 
 @interface TextileImageView ()
```

Comment:
Lots of deleted code in here because I had previously copied and modified Facebook's code, and now I just wrote it from scratch to make it our own and more simple.